psi4/driver/procrouting/dft_funcs/dict_builder.py

```diff
@@ -172,8 +172,8 @@ def check_consistency(func_dictionary):
         raise ValidationError("SCF: No exchange specified in functional %s." % (name))
 
     # 1c) require at least an empty correlation functional entry or C_MP2
-    elif "c_functionals" not in func_dictionary and "c_mp2" not in func_dictionary:
-        raise ValidationError("SCF: No correlation specified in functional %s." % (name))
+    # elif "c_functionals" not in func_dictionary and "c_mp2" not in func_dictionary:
+    #     raise ValidationError("SCF: No correlation specified in functional %s." % (name))
```

Comment:
corl_func has to do with omega?

Reply:
The idea of this check was to ensure that either we pass a LibXC `xc_functionals` in, or we pass in both correlation and exchange parts - the correlation (or exchange) can be an empty dictionary, but it has to be explicit to make sure people just didn't forget it. Not sure why you decided to disable it?